partitions.go

```diff
@@ -69,9 +70,9 @@ func (p *partitions) getAndCacheInactivePartition() (string, error) {
 		return "", err
 	}
 
-	if active == p.rootfsPartA {
+	if resolveLink(active) == p.rootfsPartA {
```

Comment:
resolveLink could just as easily be called in getRootCandidateFromMount.

Reply:
True, but I think it's a bit clearer if they stay here.

Besides, to address your comment about different systems returning different results for the mount device, could we not simply check both here? Aka:

```
if active == p.rootfsPartA || resolveLink(active) == p.rootfsPartA {
```

Reply:
I originally had this but it doesn't add any value.  All four possible combinations of links and paths are currently handled.  Basically if either active or  p.rootfsPartX are links they are reduced down to /dev/sdX. All comparisons are done at this base level.

Reply:
Ah right, I see it now.